<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HW0137" name="Test-class test 1" tests="1" failures="1" timestamp="24 Oct 2016 10:08:22 GMT" time="12.935" errors="0">
  <testcase name="c92_HOOQ_EmailLogin" time="7.222" classname="tv.hooq.web.scripts.C92_HOOQ_EmailLogin">
    <failure type="java.lang.AssertionError" message="Step Enter value Failed with description Value: &amp;lt;b&amp;gt;cigniphacti01@yopmail.com&amp;lt;/b&amp;gt; is &amp;lt;b&amp;gt; NOT &amp;lt;/b&amp;gt; entered in: Email Address expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Step Enter value Failed with description Value: <b>cigniphacti01@yopmail.com</b> is <b> NOT </b> entered in: Email Address expected [true] but found [false]
	at com.ctaf.support.HtmlReportSupport.onFailure(HtmlReportSupport.java:780)
	at com.ctaf.utilities.Reporter.failureReport(Reporter.java:92)
	at com.ctaf.accelerators.ActionEngine.type(ActionEngine.java:394)
	at tv.hooq.web.workflows.HOOQWorkflows.verifyLoginWithExistingAndInvalidEmails(HOOQWorkflows.java:83)
	at tv.hooq.web.scripts.C92_HOOQ_EmailLogin.c92_HOOQ_EmailLogin(C92_HOOQ_EmailLogin.java:31)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- c92_HOOQ_EmailLogin -->
</testsuite> <!-- Test-class test 1 -->
